js/modules/Debug/Debug.js

```diff
@@ -382,15 +384,35 @@ window.GLPI.Debug = new class Debug {
     }
 
     refreshWidgetButtons() {
-        const server_performance_button = this.getWidgetButton('server_performance');
+        // Server performance
         const server_perf = this.initial_request.server_performance;
         const memory_usage_mio = (server_perf.memory_usage / 1024 / 1024).toFixed(2);
-        server_performance_button.find('.debug-text').text(`${server_perf.execution_time}ms | ${memory_usage_mio}mio`);
-
-        const ajax_requests_button = this.getWidgetButton('ajax_requests');
-        ajax_requests_button.find('.debug-text').text(this.ajax_requests.length);
-
-        this.getWidgetButton('requests').find('.debug-text').text('Requests');
+        const server_performance_button_label = __("$1s ms using $2s mio")
+            .replace("$1s", `${server_perf.execution_time} <span class="text-muted">`)
+            .replace("$2s", `</span> ${memory_usage_mio} <span class="text-muted">`)
+            + "</span>"
+        ;
+        this.getWidgetButton('server_performance').find('.debug-text').html(server_performance_button_label);
+
+        // Database performance
+        const sql_data = this.getCombinedSQLData();
+        const database_button_label = __("$1s in $2s ms")
+            .replace("$1s", `${sql_data.total_requests} <span class="text-muted">`)
+            .replace("$2s", `</span> ${sql_data.total_duration} <span class="text-muted">`)
+            + "</span>"
```

Comment:
As said in #14594, this is weird to display at the same place the time/weight of initial request and the sum of all SQL queries for both the initial request and all ajax requests.
I think we should hide SQL queries count. It is confusing to have an increment on every ajax request (especially when ajax notifications are active and increase the counter very often).

Reply:
Maybe the `Server performance` button/tab could be removed.
I do not think it is necessary to highlight the initial request time and memory consumption, especially since in GLPI, the majority of the execution time is often spent loading the current tab, which is done in a AJAX query.

Reply:
> I think we should hide SQL queries count. It is confusing to have an increment on every ajax request (especially when ajax notifications are active and increase the counter very often).

I kinda like seeing the request count tbh.

Maybe some endpoints like ajax notifications could simply be blacklisted ?

Reply:
Problem is not on the count but on the total time. Having both initial time in "Server performance" tab title and total SQL time in "SQL" tab title may be confusing.

Reply:
I did not said that information is useless, but as it can be found easilly in the `HTTP requests` tab, I am not sure having it highlighted here is mandatory.

Reply:
I agree with @cconard96 that it's an important metric that should stay on the bar (and in the future, GLPI may not load the maj tab through AJAX).

Reply:
Latest changes after our discussion @cedric-anne 

![image](https://github.com/glpi-project/glpi/assets/42734840/69aebc7d-cfeb-485a-ac54-b2b795bd7531)